README.md

```diff
@@ -31,13 +31,12 @@ Browse and query your MongoDB databases both locally and in the cloud using [_sc
 
 - View your MongoDB database account by [signing in to Azure](#managing-azure-subscriptions) or attaching with a connection string
 - Optionally configure the setting `mongo.shell.path` if your mongo executable is not already on your system's PATH (many of the common commands have built-in support and do not require the Mongo shell to be installed - see [Prerequisites](#prerequisites))
-- Right-click on a database and select "Connect"
 - Click on "New Mongo Scrapbook" in the tree title bar
+- Use the CodeLens to connect to a database

```

Comment:
People don't necessarily know what "CodeLens" is, I'd rather you explain the steps they take.

Reply:
Prashanth?  I approved it, but I still expect you to fix any remaining comments.